src/components/ec-table/ec-table.story.js

```diff
@@ -23,6 +23,7 @@ const columns = [
     title: 'Original amount',
     sortable: true,
     tooltip: 'This is info test',
+    type: 'currency',
```

Comment:
Could you please change the data for each row to show some currencies, e.g. `EUR 100`.

Reply:
Done.